MH-FiorellaTheme5/source/partials/_info-product.erb

```diff
@@ -0,0 +1,88 @@
+<section class="product">
+  <div class="product-container-mobile">
+    <h2 class="product-title">Flower Bouquet Victoria</h2>
+    <img/>
+    <div class="product-gallery-description">
+      <b>Description:</b>
+      <p>This beautiful bouquet with variety of colorful roses, clove pink flower, hydrangea, green vibrynimns and spiders mums in green. Wrapped in satin silk ribbon.</p>
+    </div>
+    <b class="product-bouquet">Color Bouquet:</b>
+    <div class="colors-bouquet">
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
+    <div class="qty">
+      <b>Qty:</b>
+      <select>
+        <option value="1">1</option>
+        <option value="2">2</option>
+        <option value="3">3</option>
+        <option value="4">4</option>
+        <option value="5">5</option>
+        <option value="6">6</option>
+      </select>
+      <i class="fa fa-caret-down" aria-hidden="true"></i>
+    </div>
+    <p class="product-price">$84.50</p>
+    <button class="product-button-cart">ADD TO CART</button>
+    <h3 class="title-footer">Share:</h3>
+    <div class="share-icons">
+      <div><img/></div>
```

Comment:
If you are going to insert images, add a source. If not then use a `div` and a `background-image:`